languageFactories/language-factory-core/src/main/scala/cromwell/languages/util/ImportResolver.scala

```diff
@@ -94,4 +94,8 @@ object ImportResolver {
     result
   }
 
+  object HttpResolver {
+    lazy val sttpBackend = AsyncHttpClientCatsBackend[IO]()
```

Comment:
ToL: This could be implicit already + `import HttpResolver._` and the `implicit val sttpBackend = HttpResolver.sttpBackend` wouldn't be needed (I think) but 🤷‍♂️

Reply:
If it's implicit in the object then anyone importing HttpResolver would get an implicit `AsyncHttpClientCatsBackend` in their scope.

Probably not a problem, but it felt nicer to force people to opt-in in this case